include/experimental/__p2630_bits/submdspan_mapping.hpp

```diff
@@ -176,7 +177,8 @@ layout_left::mapping<Extents>::submdspan_mapping_impl(SliceSpecifiers... slices)
                                    *this, inv_map,
     // HIP needs deduction guides to have markups so we need to be explicit
     // NVCC 11.0 has a bug with deduction guide here, tested that 11.2 does not have the issue
-    #if defined(_MDSPAN_HAS_HIP) || (defined(__NVCC__) && (__CUDACC_VER_MAJOR__ * 100 + __CUDACC_VER_MINOR__ * 10) < 1120)
+    // But Clang-CUDA also doesn't accept the use of deduction guide so disable it for CUDA alltogether
+    #if defined(_MDSPAN_HAS_HIP) || defined(_MDSPAN_HAS_CUDA)
                                    std::tuple<decltype(detail::stride_of(slices))...>{detail::stride_of(slices)...})),
```

Comment:
would `make_tuple` resolve the issue?
(don't care too much)

Reply:
I'd rather avoid even more non-marked up functions